releasenotes/notes/remove-transpile-broadcast-1dfde28d508efa0d.yaml

```diff
@@ -0,0 +1,43 @@
+---
+upgrade:
+  - |
+    Support for passing in lists of argument values to the :func:`~.transpile`
+    function is removed. This functionality was deprecated as part of the
+    0.23.0 release and is now being removed. Removing this functionality was
+    necessary to greatly reduce the overhead for parallel execution for
+    transpiling multiple circuits at once. If you’re using this functionality
+    currently you can call :func:`~.transpile` multiple times instead. For
+    example if you were previously doing something like::
+
+        from qiskit.transpiler import CouplingMap
+        from qiskit import QuantumCircuit
+        from qiskit import transpile
+
+        qc = QuantumCircuit(2)
+        qc.h(0)
+        qc.cx(0, 1)
+        qc.measure_all()
+
+        cmaps = [CouplingMap.from_heavy_hex(d) for d in range(3, 15, 2)]
+        results = transpile([qc] * 6, coupling_map=cmaps)
+
+    instead you should now run something like::
+
+        from itertools import cycle
+
+        from qiskit.transpiler import CouplingMap
+        from qiskit import QuantumCircuit
+        from qiskit import transpile
+
+        qc = QuantumCircuit(2)
+        qc.h(0)
+        qc.cx(0, 1)
+        qc.measure_all()
+
+        cmaps = [CouplingMap.from_heavy_hex(d) for d in range(3, 15, 2)]
+        results = []
+        for qc, cmap in zip(cycle([qc]), cmaps):
+            results.append(transpile(qc, coupling_map=cmap))
```

Comment:
Super minor, but this seems like an unusual pattern to suggest compared to `results = [transpile(qc, coupling_map=cm) for cm in cmap]` (and corresponding removal of the `itertools` import).

Reply:
TBH, I don't know why I did this, I think I just copied it blindly from the deprecation release note (although I wrote that too). But yeah it makes sense to update this

Reply:
Updated in: https://github.com/Qiskit/qiskit-terra/pull/10291/commits/ffd10191defea7300c659fa1c4e3aa3187be3a26